substrate/launcher/main.go

```diff
@@ -12,7 +12,7 @@ import (
 	"strings"
 	"syscall"
 
-	"github.com/mitchellh/osext"
+	"github.com/kardianos/osext"
```

Comment:
Initial `make build` was throwing errors on osext. Looking at https://github.com/mitchellh/osext the readme states the `kardianos` repo should be used. I backed the change out for now as the underlying issue was the plan9 build, not the osext dep and restricting what OS is built fixed things up.